FWCore/Framework/src/Schedule.cc

```diff
@@ -320,23 +320,35 @@ namespace edm {
             it = inserted.first;
           }
 
+          bool found = false;
           for (auto const& productIter : preg.productList()) {
             BranchKey const& branchKey = productIter.first;
+            // Ignore potential alias-for products from earlier (or other) processes
```

Comment:
My memory is it is imposible to create an EDAlias for a different process (maybe not true for SubProcess?)

Reply:
A quick look on `processEDAliases()` and `checkAndInsertAlias()` confirm that EDAlias must be created on products in the same process (I'd say even within a SubProcess, because the process name is different).

Is this just a remark, or are you suggesting that the comment should be improved?

Reply:
The comment appears to be misleading.

Reply:
How about
```cpp
// The alias-for product must be in the same process as the SwitchProducer (earlier processes or SubProcesses may contain products with same type, module label, and instance name)
```
?

Reply:
sure